compiler-rt/lib/tsan/rtl/tsan_report.cpp

```diff
@@ -93,7 +93,9 @@ static const char *ReportTypeString(ReportType typ, uptr tag) {
       return "signal handler spoils errno";
     case ReportTypeDeadlock:
       return "lock-order-inversion (potential deadlock)";
-    // No default case so compiler warns us if we miss one
+    case ReportTypeMutexCannotBeLocked:
+      return "mutex cannot be locked on this code path";
```

Comment:
The word "locked" makes me think that the report is produces when a mutex is locked. We are not locking a mutex here, but rather "holding" it. The word "held" used in the interface function name looks like a better fit here. Perhaps something like "mutex held in the wrong context"?

Reply:
Will update wording